### DISCUSSION TITLE
Discord Link Not Working

### DISCUSSION THREAD

**Discussion by SickProdigy** [meisnate12/Plex-Meta-Manager#1776] [Q&A]
Was going to join and the link isn't working.
Pretty sure it's a vanity link that's been disabled.
Would be more beneficial to have the site metamanager.wiki/discord go to a non custom link that won't go away.

**Comment by chazlarson** [comment]
Fixed in the nightly and develop wiki.

**Comment by chazlarson** [comment]
Now fixed in the main wiki.
